scripts/build_and_test.sh

```diff
@@ -6,4 +6,4 @@ cmake -H"." -DVTU11_ENABLE_TESTS=ON -DVTU11_STRICT_COMPILER=ON -B"build"
 cmake --build "build"
 
 cd build # tests need to be executed in the same dir as the test_runner, otherwise path are not working correctly
-./vtu11_testrunner
+ctest -T memcheck
```

Comment:
@phmkopp this runs the tests with `valgrind`
I use this in another project, very handy!